src/main/markdown/doc/latest/DevGuideUiBinderI18n.md

```diff
@@ -1,3 +1,6 @@
+i18n UIBinder
```

Comment:
I know I convert every title by using `===` instead of `#` But it seems it's preferable tu use `#` even for the page title in order to be consistant with the rest of the page